### ISSUE TITLE
No job variety.

### ISSUE THREAD

**Issue by WooterTheTroubleshooter** [fauxnik/DVOwnership#81]
This is a fun mod idea and for my first few hauls it seemed to go pretty good. But now at Machine Factory Town I tried to get a different job, by creating additional wagons on different tracks. I was hoping that would give me multiple orders with different destinations.

![20241030210946_1](https://github.com/user-attachments/assets/8f11c51a-b69e-417a-a5f6-a281c2fbabde)

But instead it seems there is no variety in the job generation.

**Comment by matejdro** [comment]
Yeah, it would be interesting if game would spawn multiple possible jobs, so you could pick your own destinations. But not sure if DV job engine supports having multiple mutually exclusive jobs for one wagon.
